api_app/analyzers_manager/file_analyzers/goresym.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from api_app.analyzers_manager.classes import DockerBasedAnalyzer, FileAnalyzer
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+
+logger = logging.getLogger(__name__)
+
+
+class GoReSym(FileAnalyzer, DockerBasedAnalyzer):
+    name: str = "executable_analyzer"
+    url: str = "http://malware_tools_analyzers:4002/goresym"
+    # interval between http request polling
+    poll_distance: int = 5
+    # http request polling max number of tries
+    max_tries: int = 5
+
+    def update(self) -> bool:
+        pass
+
+    def run(self):
+        binary = self.read_file_bytes()
+        fname = str(self.filename).replace("/", "_").replace(" ", "_")
+        args = ["-t", "-d", "-p", f"@{fname}"]
```

Comment:
i would add parameters for all the options, these 3 ones but also the other available ones:
```
-m <virtual address> ("manual", optional) flag will dump the RTYPE structure recursively at the given virtual address
-v <version string> ("version", optional) flag will override automated version detection and use the provided version. This is needed for some stripped binaries. Type parsing will fail if the version is not accurate.
```

Reply:
you mean I should add all the available flags as parameter options in the analyzer settings?

Reply:
yep, that allows the user to customize the execution of the tool like he was running the tool standalone